src/App.js

```diff
@@ -1,15 +1,107 @@
 import React, { Component } from 'react';
+import ContactList from './components/contact-list.js';
+import SearchBar from './components/search-bar.js';
+import Button from './components/button.js';
+import axios from 'axios';
+
+const data = {
```

Comment:
I wouldn't recommend using local variables and mix it with state. This will create a whole bunch of problems when you have multiple instances of the same component on your page. They use the same data and the values get mutated in different locations. Better exclusively keep that in the component state.